pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -132,6 +130,9 @@
 
     private final Map<String, String> properties;
 
+    private final ClassLoader frameworkClassLoader;
```

Comment:
we don't use `framework` elsewhere. shall we just call it `instanceClassLoader` or `runtimeClassLoader`?

Reply:
we can call it runtimeClassLoader